scripts/winesapos-snap.sh

```diff
@@ -0,0 +1,5 @@
+git clone https://aur.archlinux.org/snapd.git
+cd snapd
+makepkg -si
+sudo systemctl enable --now snapd.socket
+sudo systemctl enable --now snapd.apparmor.service
```

Comment:
We enable the `snapd.service` here:

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.1/scripts/winesapos-install.sh#L847

but we don't do anything with `snapd.apparmor.service`. I never heard of it. I learn something new every day. :-) You can enable it for the secure image type by enabled it here:

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-beta.1/scripts/winesapos-install.sh#L1002

Something like:

```
    if [[ "${WINESAPOS_APPARMOR}" == "true" ]]; then
        chroot ${WINESAPOS_INSTALL_DIR} systemctl enable snapd.apparmor.service
```